app/cdap/components/Replicator/Create/Content/SelectTables/index.tsx

```diff
@@ -45,6 +49,8 @@ import {
   ITableInfo,
 } from 'components/Replicator/types';
 
+const PREFIX = `features.Replication.Create.Content.SelectTables`;
```

Comment:
while you are working on this file, can you also change other hardcoded string into i18n?

Reply:
done